xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml

```diff
@@ -268,11 +268,16 @@
       <Docs>
         <param name="continuation">The action to invoke when the await operation completes.</param>
         <summary>Schedules the continuation action for the asynchronous task associated with this awaiter.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+This method is intended for compiler user rather than use directly in code.
+          ]]></format>
+        </remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="continuation" /> is <see langword="null" />.</exception>
         <exception cref="T:System.InvalidOperationException">The awaiter was not properly initialized.</exception>
```

Comment:
The unsafeOnCompleted doesnot throws InvalidOperationException, we need to remove this line.

Reply:
@Anipik What about the methods called internally by UnsafeOnCompleted? Can you please confirm if they throw this exception?

Reply:
@Anipik can you please double check if the exception is thrown by internally called methods?

Reply:
yeah sure i will take a look at the internal methods as well

Reply:
@carlossanlop I verified and the internal methods does not throw invalidOperationException

Reply:
```suggestion
```